apps/api/src/persistence/repositories/repository-for-entity.ts

```diff
@@ -100,6 +101,9 @@ export class RepositoryForEntity<TEntity extends HasAggregateId & BaseDomainMode
     async update(updatedEntity: TEntity) {
         const updatedDTO = this.#mapEntityDTOToDocument(updatedEntity.toDTO());
```

Comment:
Make a note about why the `updatedEntity` and not just a partial is provided. Note also that we don't want to override additional data in the database that isn't exposed to the model.

Reply:
done